addons/map/README.md

```diff
@@ -8,6 +8,10 @@ Various tweaks to the in-game map, including:
 - Map illumination (optional)
 - Blue Force Tracker (optional)
 
+#### Variables
```

Comment:
Can you please explain what do you mean? The markdown doesn't care up to 0-3 leading spaces, so it's down to code style and I followed the syntax of the rest of the file (the other lists [start without a space](https://raw.githubusercontent.com/freghar/ACE3/f930f58732a8a/addons/map/README.md)).

If you mean it should have a leading space because it's in the Variables subsection, then it's not right either - the Markdown syntax and indeed many other wiki syntaxes define levels without indent, which exists only in lists (beyond the first level).

If you don't want it there, I can remove it, though I don't think such a minor feature deserves a separate web page and a similar syntax [is used in ACE3 already](https://github.com/acemod/ACE3/blob/master/addons/attach/README.md).

Thanks.